libs/labelbox/src/labelbox/schema/ontology.py

```diff
@@ -73,6 +74,7 @@ class Tool:
         classifications: (list)
         schema_id: (str)
         feature_schema_id: (str)
+        attributes: (list)
     """
```

Comment:
I noticed this Tool class doesn't reference a __post_init() method, which is where I print the attribute experimental warning. Should I add one?

Reply:
yes

Reply:
Added 👍